docs/docs/hydroflow/quickstart/setup.md

```diff
@@ -99,7 +99,8 @@ Client live!
 Now you can type strings in the client, which are sent to the server, echo'ed back, and printed at the client. E.g.:
 ```console
 Hello!
-2023-06-01 00:19:53.906635 UTC: Got Echo { payload: "Hello!", ts: 2023-06-01T00:19:53.906123Z } from 127.0.0.1:61019```
+2023-06-01 00:19:53.906635 UTC: Got Echo { payload: "Hello!", ts: 2023-06-01T00:19:53.906123Z } from 127.0.0.1:61019
```

Comment:
Is this going to change on every run?

Reply:
Oh no, that file is still static, just fixed a little newline issue there

There are some other logs that won't update either in the last two networked quickstart examples